lib/routes/airlevel/levelrank.ts

```diff
@@ -0,0 +1,69 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import { load } from 'cheerio'; // 类似 jQuery 的 API HTML 解析器
+
+export const route: Route = {
+    path: ['/rank/:status', '/rank'],
```

Comment:
If `status` is optional, please use [proper optional syntax](https://hono.dev/docs/api/routing#optional-parameter) instead of setting path as an array of string.